tdesign/src/Icon/IconPencil.tsx

```diff
@@ -0,0 +1,8 @@
+import { pencilIconURI } from "./cssSvgStrings";
+import BaseIcon, { IIconProps } from "./BaseIcon";
+
+const IconPencil = (props: IIconProps) => {
+  return <BaseIcon iconSlug={"pencilGray"} svgURI={pencilIconURI} {...props} />;
+};
+
+export default IconPencil;
```

Comment:
Long term question, not related to this MR, probably involving @milesrichardson:

Do you think we should be using the same pattern for consistency and use `export default` here even though we're shying away to named exports?

Personally, I would start using named exports in new code, which could also simplify the `index.tsx` in this directory (we could use `export * from './IconPencil'; instead of the existing 2 liner)

Reply:
Sure, it makes sense to change this for consistency with the approach in the rest of the codebase.

The icon files are not currently manually edited, btw (at least not any that existed before this PR). They were all generated with some scaffolding code (there is a bash script called `mkrci` in the parent repo). So if we change them, we should (a) change every existing icon file, and (b) change the scaffolding code.

That would make sense to do in a separate PR – I made a ClickUp. (#23k1cpq)

Reply:
Nice, thanks!